spec/lib/api/v3/cost_types/cost_type_representer_spec.rb

```diff
@@ -104,21 +104,21 @@
   context 'generation' do
     subject(:generated) { representer.to_json }
 
-    it { should include_json('CostType'.to_json).at_path('_type') }
+    it { is_expected.to include_json('CostType'.to_json).at_path('_type') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.